### ISSUE TITLE
Level tiers

### ISSUE THREAD

**Issue by bviktor** [Sinewyk/kf2_autokick#2]
It'd be pretty sweet if we could specify tiers, that is, minimum perk levels for each difficulty. Essentially you'd have to find the line right after `<dt>Difficulty</dt>`. Do you think that's possible to implement?

**Comment by Sinewyk** [comment]
Hmm, just launch another process with a different conf ?

I mean, I get why you would want that. But I do know it's not *necessary* to have what you want =)

**Comment by bviktor** [comment]
The problem with your logic is that changing the difficulty only requires webadmin while changing to a different config requires SSH which in turn requires my physical keychain, i.e. major inconvenience.

Yeah, it's not "necessary", just very useful.

**Comment by Sinewyk** [comment]
Yeah sorry, I was thinking about the "You have 2 servers, one in suicidal, the other in Hoe", just launch two processes.

I did not think about "You have one server that you quickly switch from HoE to hard", but because the minLevel is still set to 20 well ... not very practical.

True. I'll think about it.
